lib/jxl/jpeg/jpeg_data.cc

```diff
@@ -370,6 +370,38 @@ Status JPEGData::VisitFields(Visitor* visitor) {
     }
   }
 
+  {
+    size_t dht_index = 0;
+    size_t scan_index = 0;
```

Comment:
dht_index and scan_index don't seem to be changed in the loop

Reply:
Whoa! Fixed.